scripts/motion_planner.py

```diff
@@ -9,47 +13,92 @@ class MotionPlanner:
     def __init__(self, _action_server):
         self._action_server = _action_server
         self.task = None
+        self.velocity_pub = rospy.Publisher('movement_velocity_targets', TwistStampedArrayStamped, queue_size=0)
+        self.arm_service = rospy.ServiceProxy('uav_arm', SetBool)
```

Comment:
I ended up rolling it into a fix on that branch. I expect to make a PR for the safety client tonight.